generator-mediator-js/app/templates/mediatorTemplate/test/_openhim.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
```

Comment:
This file can be removed as well. Perhaps rather rather include a sample mock unit test just to show new developer how a test would be structured.